src/components/DataChart/DateSelector.tsx

```diff
@@ -18,14 +17,9 @@ export const periods: Record<DataRange, keyof HistoricalData<DataEntry>> = {
 } as const;
 
 export function DateSelector(props: Props) {
-  const { range, setRange, fullDomain } = props;
+  const { range, setRange, availableRanges } = props;
```

Comment:
Nit can do this instead of setting one variable equal to another: 
```suggestion
  const { range, setRange, availableRanges: options } = props;
```